demos/helloasyncio/helloasyncio.py

```diff
@@ -0,0 +1,30 @@
+import tornado.httpserver
+import tornado.ioloop
+import tornado.options
+import tornado.web
+
+from tornado.options import define, options
+
+from tornado.ioloop import IOLoop
+IOLoop.configure('tornado.platform.asyncio.AsyncIOLoop')
```

Comment:
Sure, that was a quick snippet for testing auto-reload feature, I've just extended it to highlight tricky things!